### PR TITLE
[FEATURE] show status sign on ticketDetails at the title

### PR DESCRIPTION
**Pull request by sarykemmoun**
added a screenshot with the title and the new feature marked.

screenshot from 2017-10-22 16-53-14

### CODE REVIEW

File: js/components/ApprovalsDetails/Description.js~HEAD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        marginBottom: 20,
+        fontWeight: 'bold'
+    },
+    contentStyle: {
+        color: black,
+        fontStyle: content ? 'normal' : 'italic'
+    }
+});
+
+export default Description;
```
Comment (raphaelboukara) [inline js/components/ApprovalsDetails/Description.js~HEAD]: ?

File: js/components/TicketDetails/Description.js~HEAD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    description: {
+        marginBottom: 20,
+        fontWeight: 'bold'
+    },
+    contentStyle: {
+        color: black,
+        fontStyle: content ? 'normal' : 'italic'
+    }
+});
+export default Description;
```
Comment (raphaelboukara) [inline js/components/TicketDetails/Description.js~HEAD]: ????????

File: js/components/TicketDetails/TitleContainer.js [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+const ticket = tickets[ticketId];
+const { title, creationDate, lastModifiedDate, assignedTo, status } = ticket;
+return {
+title,
+creationDate,
+text2: I18n.t('modified'),
+date2: lastModifiedDate,
+assignedTo,
+status: status.charAt(0).toUpperCase()
+};
```
Comment (raphaelboukara) [inline js/components/TicketDetails/TitleContainer.js]: indentation

**Comment by raphaelboukara** [comment]
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️   [FEATURE] + [REFACTOR] right? ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️

**Comment by sarykemmoun** [comment]
@raphaelboukara this is a sub-branch.
the parent branch is a [REFACTOR].
but this is only a [FEATURE]

**Pull request closed without merging**